.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+## Requirements for PRs to the SSO-Dashboard.
```

Comment:
does this popup when you try to PR?
Else, would prefer linking to github.com/mozilla-iam/mozilla-iam and have these requirements there (which it already has for the most part i think)

Reply:
Yes it does pop up every time you PR.  I thought it was better to put the requirements front and center instead of sending someone to yet another web page to know what needed to happen in order to land a pull request.  

The language is also slightly different.

Reply:
we should sync these IMO (either way, I dont mind if you copy this back to mozilla-iam nearly verbatim). Maybe we could also just drop the file there so that we can copy/paste it for other projects easily

Reply:
Should likely put a pull request template and an issue template in mozilla-iam/mozilla-iam